### ISSUE TITLE
Settings are reset

### ISSUE THREAD

**Issue by skjetne** [sirWest/AudioSwitch#119]
As of lately my settings keeps getting reset. Happened around win 10 creators update, but not 100% certain. I downloaded the latest version last week, and still have the issues. I'm on windows 10 v. 1703 (15063.296).

**Comment by bharding512** [comment]
This happens to me every time Windows 10 has a significant update

**Comment by ghost** [comment]
I have the same issue. I copied the settings and leave the copy in the AppData/Local folder. Whenever it's gone, I replace the new one with it.

**Comment by beppe9000** [comment]
I am experiencing this on Windows Server 2016 between reboots.

**Comment by FuzzyFoxe** [comment]
This is still an issue in 2019, there have been no updates to address this issue thus far. Has this program been abandoned?
P.S. Thanks for helping me find the settings file, @jesario :D

**Comment by TurturOk** [comment]
Still an issue in 2023

**Comment by slessardjr** [comment]
> Still an issue in 2023

It seems this project has been abandoned. The last commit was like 6 years ago.

**Comment by TurturOk** [comment]
@slessardjr you're right, I didn't realise. I guess I'm happy that's the only issue bugging me right now in an app no longer being maintained. Will have to find an alternative eventually.
